client/src/locales/fr-FR.json

```
@@ -955,7 +955,13 @@
         },
         "status": {
             "active": "Actif",
-            "revoked": "Révoqué"
+            "revoked": "Révoqué",
+            "frozen": "Suspendu"
+        },
+        "statusDescriptions": {
+            "active": "L'utilisateur est actif.",
```

I'd be more explicit here even if it is a little redundant
```suggestion
            "active": "L'utilisateur est actif et peut accéder normalement à cet organisation.",
```